.travis.yml

```diff
@@ -62,10 +62,27 @@ script:
       else
         export NPROC=1;
       fi;
-    cmake . -DCMAKE_INSTALL_PREFIX=/usr -DWANT_HUNSPELL=1 -DWITH_PODOFO=1 -DWANT_GRAPHICSMAGICK=1 -DWANT_DEBUG=0 -DWANT_SVNVERSION=0 -DWANT_GUI_LANG=en_US;
-    make -j"$NPROC";
-    make install;
-    bash -ex ./AppImage-package/bundle.sh;
+      cmake . -DCMAKE_INSTALL_PREFIX=/usr -DWANT_HUNSPELL=1 -DWITH_PODOFO=1 -DWANT_GRAPHICSMAGICK=1 -DWANT_DEBUG=0 -DWANT_SVNVERSION=0 -DWANT_GUI_LANG=en_US;
+      make -j"$NPROC";
+      make DESTDIR=appdir -j"$NPROC" install ; find appdir/;
+      cp /usr/lib/python2.7/lib-dynload/_tkinter.so appdir/usr/;
+      cp -r /usr/lib/tcltk appdir/usr/lib/;
+      cp -r /usr/share/tcltk appdir/usr/share/;
+      sed -i -e 's|/usr/share/scribus|././/share/scribus|g' appdir/usr/bin/scribus;
+      sed -i -e 's|/usr/lib/scribus|././/lib/scribus|g' appdir/usr/bin/scribus;
+      sed -i -e 's|/usr/share/doc/scribus/|././/share/doc/scribus/|g' appdir/usr/bin/scribus;
+      wget -c -nv "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage";
+      chmod a+x linuxdeployqt-continuous-x86_64.AppImage;
+      unset QTDIR; unset QT_PLUGIN_PATH ; unset LD_LIBRARY_PATH;
+      export VERSION=$(git rev-parse --short HEAD);
+      ./linuxdeployqt-continuous-x86_64.AppImage appdir/usr/share/applications/*.desktop -bundle-non-qt-libs;
+      ./linuxdeployqt-continuous-x86_64.AppImage --appimage-extract;
+      mv appdir/usr/_tkinter.so appdir/usr/lib/;
+      ( cd appdir/usr/ ; ln -s ./lib/_tkinter.so . );
+      ./squashfs-root/usr/bin/linuxdeployqt -bundle-non-qt-libs appdir/usr/lib/_tkinter.so;
+      cp /opt/qt58/plugins/iconengines/libqsvgicon.so appdir/usr/plugins/;
```

Comment:
Never manually copy stuff into the AppImage. This does not properly deploy its dependencies, and does not set the rpath. Instead, pass in `-extra-qt-plugins` as described here: https://github.com/probonopd/linuxdeployqt/#adding-extra-qt-plugins